### PR TITLE
Basic structure-milestone-1

### PR DESCRIPTION
**Pull request by ahmetbozaci**
**In this milestone,**
- Use gitflow
- Use webpack
- Add basic layout 

Screenshot from 2021-08-03 16-35-10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Leaderboard
 
-> One paragraph statement about the project.
+> CHANGE IT
 
-![screenshot](./app_screenshot.png)
+![screenshot](./screenshot.png)
```
Comment (vichuge) [inline README.md:7]: - [x] It looks like you have an issue with your image. I kindly invite you to fix it. 🐶 
image

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+      </div>
+      <div>
+        <h3>Add your score</h3>
+        <form action="#">
+          <input type="text" placeholder="Your name" /><br>
+          <input type="text" placeholder="Your score" /><br>
+          <button type="submit">Submit</button>
+        </form>
+      </div>
+    </div>
```
Comment (vichuge) [inline src/index.html]: - [ ] Please, in place to use HTML code to the code, I invite you to create the project with js code, preferable inside  `src/index.js`. Try this, on this script use `<div id="root"></div>`, after that use index.js to write all this structure. 🐱 
- Don't forget to follow the ES6 best practices when you do it.
- `index.html main contain only root element`

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] On this project, is not required to use the HTML-CSS Linters config (is enough with js). I invite you to fix this following the js [linters template](https://github.com/microverseinc/linters-config/tree/master/javascript). 🐦 
image


- [x] Check the comments under the review.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ahmetbozaci) [reply README.md:7]: This is first milestone-1 project is not completed in this milestone so I didn't add it. But I will add. Thank you.

File: src/index.html (hunk 2 above)
Comment (ahmetbozaci) [reply src/index.html]: @vichuge I can't see this in project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md) Is it good practise create everything in js file?

Comment (mimipeshy) [reply src/index.html]: @ahmetbozaci hello there, I would like to clarify that what you did was enough for this milestone but if you created it in js would still be fine. So don't worry about it 😄

**Review by mimipeshy — APPROVED** [review]
### Hi @ahmetbozaci 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: src/index.html (hunk 2 above)
Comment (ahmetbozaci) [reply src/index.html]: @minipeshy Thank you.

**Pull request merged**